snap/snapcraft.yaml

```diff
@@ -21,9 +23,16 @@ parts:
     prepare: DMD=../../../stage/bin/dmd ./build.sh
     stage:
     - bin/dub
+    - lib/lib*.so*
+    - lib/x86_64-linux-gnu/lib*.so*
+    - usr/lib/lib*.so*
+    - usr/lib/x86_64-linux-gnu/lib*.so*
+    - etc/ldap/*
```

Comment:
These extra `stage` entries are necessary if we want to use `stage-packages` with the `dump` plugin.  But there's an easier way: instead of adding all these individual (potentially flimsy) choices, add a separate part:

```
curl:
  plugin: nil
  stage-packages: libcurl3-gnutls
```

Taken together with the `LD_LIBRARY_PATH` environment settings, this ought to work otherwise identically.